types/titanium/index.d.ts

```diff
@@ -16880,11 +18252,6 @@ declare namespace Titanium {
 			 */
 			getHtml(): string;
 
-			/**
-			 * Sets the value of the <Titanium.UI.WebView.html> property.
-			 */
-			setHtml(html: string): void;
```

Comment:
This shouldn't be removed. See https://github.com/appcelerator/titanium_mobile/blob/master/apidoc/Titanium/UI/WebView.yml#L145, this is not just a simple setter. Not sure if this is an issue with the TS definition generator or the docs parser, though.

Reply:
I think this one was a dupe of the one with both the html and options parameters on [line 18041](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34512/files#diff-57636c23872b22c51a9183be5b85f9f9R18041). For methods like this that matched a setter but also had a declared method the doc system would dupe them, [this commit](https://github.com/appcelerator/titanium_mobile/commit/b5d6124918bf1bb12981ffaf6b2cb7c46741b2b5) fixed that

Reply:
Ah, perfect!